Mathlib/Algebra/Order/Module/Synonym.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Module.Basic
+import Mathlib.Algebra.Order.Pi
+import Mathlib.Algebra.Ring.OrderSynonym
+
+/-!
+# Action instances for `OrderDual`
+
+This file provides instances of algebraic actions for `OrderDual`. Note that the `SMul` instance is
```

Comment:
```suggestion
This file provides instances of algebraic actions for `OrderDual`. Note that the `SMul` instances are
```
since there are two of them

Reply:
Damn! Just past the rewrapping point.